services/movie_session.py

```diff
@@ -42,3 +45,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    return list(
+        Ticket.objects
+        .filter(movie_session_id=movie_session_id)
```

Comment:
Or reuse get_movie_session_by_id and use related_name to fetch tickets of this movies session. But this is ok as well